oep-template.rst

```diff
@@ -0,0 +1,85 @@
+=================================
```

Comment:
Does this template need an explanation? Or will I find that elsewhere? You can add comments in the doc if you want:
```
.. hey there!
```